src/core/JsonConfigCommandCenter.ts

```diff
@@ -7,9 +7,11 @@ import { DigitalTwinApi } from "./DigitalTwinApi";
 import { Api } from "./Api";
 import { LOCALIZATION } from "../constants";
 import { JSONEditorMode } from "jsoneditor";
+import hash from 'object-hash';
```

Comment:
Can we match the pure strings to check the integrity? (since hash generation needs additional dependency)

Reply:
Yeah I saw both works. Then I'll use string comparison